user_one.rb

```diff
@@ -0,0 +1,21 @@
+module MyEnumerable
+  def all?
+    each { |item| return false unless yield(item) }
+    true
+  end
+end
+
+class MyList
+  include MyEnumerable
+
+  def initialize(list)
+    @list = list
+  end
+
+  def each(&block)
+    @list.each(&block)
+  end
+end
```

Comment:
- [ ] It would be a good thing to make these two to be in separate files. This way, you will have separated the logic and will not lead to any confusion.